### PR TITLE
PT-2848: Menu options on Family page omitted in translation

### PR DESCRIPTION
**Pull request by veronikaslc**

### CODE REVIEW

File: components/skin/api/src/main/resources/ApplicationResources_es.properties [hunk 1]
```diff
@@ -12,6 +12,8 @@ phenotips.contentTopMenu.moreActionsMenu.history.title=Ver c\u00f3mo el registro
 phenotips.contentTopMenu.moreActionsMenu.exportPDF=Exportar a PDF
 phenotips.contentTopMenu.moreActionsMenu.exportXAR=Exportar como XAR
 phenotips.contentTopMenu.moreActionsMenu.exportJSON=Exportar a JSON
+phenotips.contentTopMenu.familyActionsMenu.deleteAndUnlink=Eliminar la familia y los miembros de desvincular
```
Comment (sdumitriu) [inline components/skin/api/src/main/resources/ApplicationResources_es.properties]: This seems wrong to me...

Comment (sdumitriu) [inline components/skin/api/src/main/resources/ApplicationResources_es.properties]: `Eliminar la familia y deconectar los miembros` perhaps?

File: components/skin/api/src/main/resources/ApplicationResources_es.properties [hunk 2]
```diff
@@ -12,6 +12,8 @@ phenotips.contentTopMenu.moreActionsMenu.history.title=Ver c\u00f3mo el registro
...
+phenotips.contentTopMenu.familyActionsMenu.deleteAllMembers=Borrar todos los miembros de la familia y
```
Comment (sdumitriu) [inline components/skin/api/src/main/resources/ApplicationResources_es.properties]: This seems wrong as well. `Eliminar` used above and in most other translations, `Borrar` here. How about:

    Eliminar la familia y todos los miembros

Comment (veronikaslc) [inline components/skin/api/src/main/resources/ApplicationResources_es.properties]: I totally don't speak Spanish :) What I do is I just use google translate

**Comment by veronikaslc** [comment]
commit amended

**Pull request merged**